### PR TITLE
18.x php update helper

### PR DESCRIPTION
**Pull request by JedMeister**
Minor tweaks to @marcos-mendez's contribution (in #290) - just changes as suggested by shellcheck & a visual adjustment. Plus also adding acknowledgement (that will show up in all relevant changelogs).

Thanks again @marcos-mendez for this awesome contribution. A number of users have been screaming out for this - so you'll make their day. I will get this into the next batch of v18.0 appliances and all future appliances will include it. As PHP v8.3 has only been out a few months, my guess is we won't need to tweak it for v8.4 for some time (I forget their release schedule).

Also are you happy with that acknowledgement line @marcos-mendez? I.e.:
```
[Marcos Méndez @ POPSOLUTIONS <https://github.com/marcos-mendez>]
```

Or would you prefer me include an email address? Any other adjustments?

### CODE REVIEW

**Comment by marcos-mendez** [comment]
LGTM 😎

File: overlays/php/usr/local/bin/tkl-upgrade-php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# Function to display PHP version choices and get user input
+choose_php_version() {
+    echo "Select the PHP version to install:"
+    echo "1) PHP 7.3"
+    echo "2) PHP 7.4"
+    echo "3) PHP 8.0"
+    echo "4) PHP 8.1"
+    echo "5) PHP 8.2"
+    echo "6) PHP 8.3"
+    read -p -r "Enter choice [1-5]: " php_choice
```
Comment (OnGle) [inline overlays/php/usr/local/bin/tkl-upgrade-php:12]: I think this should be `[1-6]`

**Review by OnGle — CHANGES_REQUESTED** [review]
Tiny little change. Otherwise lgtm, thanks for the awesome contribution! :smile:  :partying_face:

File: overlays/php/usr/local/bin/tkl-upgrade-php (hunk 1 above)
Comment (JedMeister) [reply overlays/php/usr/local/bin/tkl-upgrade-php:12]: Good catch man!

**Comment by marcos-mendez** [comment]
My local version is 1 to 6 altought i didnt tested it but it LGTM

**Comment by JedMeister** [comment]
I'll sort that minor mistake/typo and then merge. Thanks @OnGle & thanks again @marcos-mendez :grin:

**Pull request merged**